src/types.rs

```diff
@@ -40,6 +40,9 @@ pub enum Change {
         versions: Vec<CrateVersion>,
     },
     /// A crate version was deleted.
+    ///
+    /// Note that this is equivalent to deleting a line from a crates version file.
```

Comment:
It's all good here with the `VersionDeleted` test, @syphar, but I want to bring to your attention that multiple versions deleted in the same commit will be non-deterministic in order. That shouldn't matter, but I wanted to point it out in case that is the next lurking bug that will hit when we least expect it 😅.

Reply:
thanks for the update here! I'm happy the test is fine. 

From what I see, the order doesn't matter, also it doesn't have to be stable.